pytest_selenium/driver.py

```diff
@@ -38,31 +39,43 @@ def browserstack_driver(item, capabilities):
 
 def chrome_driver(item, capabilities):
     """Return a WebDriver using a Chrome instance"""
-    return webdriver.Chrome(desired_capabilities=capabilities)
+    options = item.config.option
+    kwargs = {'desired_capabilities': capabilities}
+    if options.driver_path is not None:
+        kwargs['executable_path'] = options.driver_path
+    return webdriver.Chrome(**kwargs)
 
 
 def firefox_driver(item, capabilities):
     """Return a WebDriver using a Firefox instance"""
     options = item.config.option
-    binary = None
-    if options.firefox_path:
+    kwargs = {'capabilities': capabilities or None}
```

Comment:
Selenium explicitly handles `None`, the bug here is that `chrome_driver` should probably include `or None`.